viewer/src/utilities/spinnerStyles.css

```diff
@@ -0,0 +1,37 @@
+.reveal-viewer-spinner,
+.reveal-viewer-spinner:after {
+    border-radius: 50%;
+    width: 32px;
+    height: 32px;
+}
```

Comment:
Is this 'mobile-friendly'?

Reply:
Yep. It's still a small little spinner at the top corner.